src/main/java/com/todolist/app/springmvcboottodolist/controllers/HelloController.java

```diff
@@ -0,0 +1,13 @@
+package com.todolist.app.springmvcboottodolist.controllers;
+
+import ch.qos.logback.core.model.Model;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class HelloController {
+    @GetMapping("/hello")
+    public String hello(Model model){
```

Comment:
Can we implement a small test for this controller, e.g. see https://spring.io/guides/gs/testing-web/?